python/pp_ser.py

```diff
@@ -659,6 +638,7 @@ def __re_endmodule(self):
             if self.__module != m.group(2):
                 self.__exit_error(msg = 'Was expecting "end '+m.group(1)+' '+self.__module+'"')
             self.__module = ''
+            self.__useStmtInModule = False
```

Comment:
Where is the use statement in the module actually placed? Am I missing something?